jujugui/static/gui/src/app/components/shared/basic-table/row/row.js

```diff
@@ -145,12 +144,13 @@ BasicTableRow.propTypes = {
   // A function to call when a row is clicked.
   clickURL: PropTypes.string,
   columns: PropTypes.arrayOf(PropTypes.shape({
-    content: PropTypes.node,
-    // The number of columns (between 1 and 12).
-    columnSize: PropTypes.number.isRequired,
-    // The extra classes to apply to the column.
-    classes: PropTypes.arrayOf(PropTypes.string)
-  }).isRequired).isRequired,
+      content: PropTypes.node,
+      // The number of columns (between 1 and 12).
+      columnSize: PropTypes.number,
```

Comment:
I don't think this is required in the cell where it's used.

Reply:
It's not - I've removed.